lms/static/js/spec/financial-assistance/financial_assistance_form_view_spec.js

```diff
@@ -36,10 +94,48 @@ define([
                     dashboard_url: '/dashboard',
                     platform_name: 'edx',
                     submit_url: '/api/financial/v1/assistance'
-                };
+                },
+                completeForm = {},
```

Comment:
No need to assign this a temporary value, since it's being reassigned a few lines down.